airbyte-workers/src/test/java/io/airbyte/workers/protocols/airbyte/DefaultAirbyteDestinationTest.java

```diff
@@ -147,14 +147,4 @@ public void testCloseNotifiesLifecycle() throws Exception {
     verify(outputStream).close();
   }
 
-  @Test
```

Comment:
I got rid of these test because I didn't feel there gave us much.

As is, the cancel operation was resulting in exit values of 1 and 127 and resulting in exception even though the system was doing as instructed. 127 is the general sig term exit code and is expected. The meaning of the 1 exit code depends on the operating system. 

Regardless of exit code, my tests showed that the containers/processes exited cleanly.